apps/expo/src/components/Discover/DiscoverHomeCard.tsx

```diff
@@ -14,32 +14,37 @@ interface Props {
 
 const DiscoverHomeCard: FC<Props> = (props) => {
```

Comment:
can you rename the folder to `/components/discover/DiscoverHomeCard.tsx`. Make it lower case.